src/utils.c

```diff
@@ -562,3 +562,46 @@ gchar *r_prepare_env_key(const gchar *key, GError **error)
 
 	return g_steal_pointer(&result);
 }
+
+gboolean r_update_symlink(const gchar *name, const gchar *target, GError **error)
```

Comment:
Wouldn't

```suggestion
gboolean r_update_symlink(const gchar *target, const gchar *name, GError **error)
```

be more intuitive for this since it matches the argument order of `symlink()`?

Reply:
OK. Will change.